src/test/regress/expected/fkeys_between_local_ref.out

```diff
@@ -385,6 +385,28 @@ BEGIN;
  reference_table_1 | n          | t
 (8 rows)
 
+ROLLBACK;
+BEGIN;
+  -- disable foreign keys to reference tables
+  SET LOCAL citus.enable_local_reference_table_foreign_keys TO false;
+  CREATE TABLE local_table_6 (col_1 INT PRIMARY KEY);
+  -- first convert local tables to citus local tables in graph
+  ALTER TABLE local_table_2 ADD CONSTRAINT fkey_11 FOREIGN KEY (col_1) REFERENCES reference_table_1(col_1) ON DELETE CASCADE;
+  CREATE TABLE local_table_5 (
+    col_1 INT UNIQUE REFERENCES local_table_6(col_1),
+    col_2 INT REFERENCES local_table_3(col_1),
+    FOREIGN KEY (col_1) REFERENCES local_table_5(col_1));
+  -- Now show none of local_table_5 & 6 to should be converted to citus local tables
```

Comment:
maybe:
-- Now show none of local_table_5 & 6 is converted to a citus local table